nixos/modules/services/web-apps/nextcloud.nix

```diff
@@ -1,12 +1,17 @@
-{ config, lib, pkgs, ... }:
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
```

Comment:
Could you please make the formatting changes in a different commit to logic changes (as a general advice for future PRs)? It makes it easier to review.

Reply:
oh yes of course. just applied the usual habbit of executing `nixfmt`. I reverted the changes not related to my lines.